applications/StructuralMechanicsApplication/python_scripts/python_solvers_wrapper_structural.py

```diff
@@ -74,24 +74,10 @@ def CreateSolverByParameters(model, solver_settings, parallelism):
         kratos_module = "KratosMultiphysics.ContactStructuralMechanicsApplication"
         solver_module_name = "contact_" + solver_module_name
 
-        if solver_settings.Has("use_computing_model_part"):
-            if not solver_settings["use_computing_model_part"].GetBool():
-                KratosMultiphysics.Logger.PrintWarning('MechanicalSolver', 'Contact Problems must use the computing model part for now, setting it to on')
-        else:
-            solver_settings.AddEmptyValue("use_computing_model_part")
-        solver_settings["use_computing_model_part"].SetBool(True)
-
     elif solver_settings.Has("mpc_contact_settings"):  # This is a mpc contact problem
         kratos_module = "KratosMultiphysics.ContactStructuralMechanicsApplication"
         solver_module_name = "mpc_contact_" + solver_module_name
 
-        if solver_settings.Has("use_computing_model_part"):
-            if not solver_settings["use_computing_model_part"].GetBool():
-                KratosMultiphysics.Logger.PrintWarning('MechanicalSolver', 'Contact Problems must use the computing model part for now, setting it to on')
-        else:
-            solver_settings.AddEmptyValue("use_computing_model_part")
-        solver_settings["use_computing_model_part"].SetBool(True)
```

Comment:
Don't you want to set it to false?

Basically what is done a few lines later for regular structural problems

Reply:
I tried, and it fails. I have an expected structure of model parts, if not followed it fails. It is everything or nothing :P